src/pages/index.js

```diff
@@ -174,15 +170,18 @@ function handleAddCardFormSubmit(cardData) {
     });
```

Comment:
`            cardDeleteModal.setLoading(false);
`

You need to return the **default** button text only in blocks `finally` so that the button could get the text back in any case.